### ISSUE TITLE
Add a button to check status of printer [hot end temp, bed temp, fan speed]

### ISSUE THREAD

**Issue by chapimp** [makerbase-mks/MKS-TFT#2]
is there a way to add our own buttons and get status of printer

**Comment by crbertilson** [comment]
Yes I agree. It would be great if we could customize the buttons. Having one display with the status of the bed, hotend, fan, and other information would be extremely useful.

**Comment by smots** [comment]
MKSTFT32 does not display the temperature of the first extruder when using two extruders. But it all works. How to fix it ?
![img_0207 1](https://cloud.githubusercontent.com/assets/18596760/14711084/ae1f4efa-07e0-11e6-9e4a-bcf8f09cee79.JPG)

**Comment by Misterblue31** [comment]
Hello, I also 2 extruders use my I3, with the TFT screen. For the E1 temp displays 0 ° measured, you have to use the descriptor E1 T0 in config.txt of your smoothie firmware (eg temperature_control.hotend.designator           T0           # )
If that's the case, I encountered same problem. 
Your designator is not wrong, but the MKS TFTv1.2 firmware does not know the T0 designator, it only recognizes T for E1 (T 0, delete and you should not have any problem).
The problem is that now the E1 temp value will be displayed correctly on your TFT, but it will wrong with slic3r. 
This slicer uses the designator T as generic designator for a single extruder. When using multiple extruders with Slis3r, everyone should have a number in the config.txt smoothie (eg T0 for E1, T1 for E2 ...)
Ideally, the MKS TFTv1.2 firmware or corrected so that the temp designator T or T0 for E1

**Comment by smots** [comment]
Thank you. Now it is clear.

2016-05-23 16:53 GMT+04:00 Misterblue31 <notifications@github.com>:
